spec/models/dashboard_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+require "rails_helper"
```

Comment:
Did using just "spec_helper" break? Nothing added in the test seems to cause that.